pkg/distro/fedora/distro.go

```diff
@@ -368,6 +368,24 @@ var (
 		exports:             []string{"xz"},
 		basePartitionTables: minimalrawPartitionTables,
 	}
+
+	coreosQcow2ImgType = imageType{
+		name:        "coreos-qcow2",
+		filename:    "image.qcow2",
+		mimeType:    "application/x-qemu-disk",
+		packageSets: map[string]packageSetFunc{},
+		defaultImageConfig: &distro.ImageConfig{
+			Locale: common.ToPtr("en_US.UTF-8"),
+		},
+		defaultSize:         10 * common.GibiByte,
+		rpmOstree:           true,
+		bootable:            true,
+		image:               coreosImage,
+		buildPipelines:      []string{"build"},
+		payloadPipelines:    []string{"ostree-deployment", "image", "qcow2"},
+		exports:             []string{"qcow2"},
+		basePartitionTables: iotBasePartitionTables,
```

Comment:
I probably need to dig into `iotBasePartitionTables` a little bit more to understand why they are being used here.

Reply:
I based the image off the iot-raw-image and reused that partition table.  I would say, instead of digging, point me to the default partition table you use in coreos images and I'll replicate it here.

Reply:
[this](https://github.com/osbuild/osbuild/blob/5c345fb3fa5d9fa5804ba74e7dc4445ae8ff1010/test/data/manifests/fedora-coreos-container.mpp.yaml#L2-L27) is what I've been using for now.

There is one theme here, though, that I think we keep bumping up into, which is where definitions like this should live. I think it was also touched on in https://github.com/osbuild/osbuild-deploy-container/issues/4 
      I

Reply:
For Image Builder (osbuild-composer and the service), the defaults are defined here.  For all the RHEL images we build for example as part of the composes in Brew and for all the images in c.rh.c/ib, this is the source of truth right now.